docs/release-notes/8.5.asciidoc

```diff
@@ -9,6 +9,7 @@
 [[known-issue-8.5.0]]
 ==== Known issues
 * Users might experience slightly longer installation and upgrade times for the user and host risk score features ({pull}142434[#142434]).
+* Version 8.5.0 {elastic-endpoint}s running on Linux systems with many CPUs may become unhealthy. For a workaround refer to https://github.com/elastic/endpoint/issues/34[issue #34].
```

Comment:
We can probably omit the version at the beginning since everything on the page is for version 8.5.0.
```suggestion
* {elastic-endpoint}s running on Linux systems with many CPUs may become unhealthy. For a workaround refer to https://github.com/elastic/endpoint/issues/34[issue #34].
```